packages/lesswrong/components/collections/CollectionsCard.jsx

```diff
@@ -17,31 +17,39 @@ const styles = theme => ({
   }
 })
 
-const CollectionsCard = ({collection, big = false, url, classes}) => {
-  const cardContentStyle = {borderTopColor: collection.color}
+class CollectionsCard extends PureComponent {
+  handleClick = (event) => {
```

Comment:
I suggest having a more detailed comment explaining what this is and why it's needed. Specifically, explain that we're trying to have an element that is clickable but also contains anchor tags.